packages/combo-box/test/toggling-dropdown.test.js

```diff
@@ -317,36 +318,6 @@ describe('toggling dropdown', () => {
     });
   });
 
-  describe('disabled', () => {
-    beforeEach(() => {
-      comboBox.disabled = true;
-    });
-
-    it('toggleIcon should not be hidden when disabled', () => {
-      expect(getComputedStyle(comboBox._toggleElement).display).not.to.equal('none');
```

Comment:
This test does not make much sense as we don't have default styles for toggle button in `src`.
The appearance of the button is controlled by the theme and covered by visual tests.